contracts/Core.sol

```diff
@@ -186,7 +186,13 @@ contract Core is CoreInterface, Util {
 	/**
 	 *	@notice Verify account proof of OpenSTRemote and commit storage root at given block height
 	 *
-	 *  @dev ProofVerificationSkipped event needed to identify replay calls for same block height
+	 *  @dev proveOpenST can be called by anyone to verify merkle proof of OpenSTRemote contract address. OpenSTRemote is OpenSTUtility
```

Comment:
Realizing this question may be out of scope (and also maybe mostly a question of my own ignorance), but why do we talk about "proving" when what's happening is "verifying" the proof? That is, shouldn't items like "proveOpenST", "wasAlreadyProved", and "provenStorageRoot" not instead be, "verifyOpenSTProof", "wasAlreadyVerified", and "verifiedStorageRoot"?

Reply:
It will be better if we discuss this point on call as writing here could be long. Will set a calendar event when you are back.